packages/help-center/src/components/help-center-sibyl-articles.tsx

```diff
@@ -81,7 +83,15 @@ export function SibylArticles( { message = '', supportSite }: Props ) {
 			>
 				{ articles.map( ( article ) => (
 					<li key={ article.link }>
-						<Link to={ getPostUrl( article as Article, message ) }>
+						<Link
+							onClick={ ( event ) => {
+								if ( 'en' !== locale ) {
+									event.preventDefault();
+									window.open( article.link, '_blank' );
+								}
+							} }
+							to={ getPostUrl( article as Article, message ) }
```

Comment:
WDYT about moving this logic into `getPostURL` then adding `target={'en' !== locale ? 'blank' : ''`. It's more accessible this way, people will know where they're going before they click.